contracts/contracts/test/MockToken.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity ^0.8.6;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract MockToken is ERC20 {
+  constructor(uint256 initialSupply) public ERC20("Mock", "MCK") {
+    _mint(msg.sender, initialSupply);
+  }
```

Comment:
Could we add an `external` mint function that anyone can call? That way we can mint during a test if needed, and it's a bit easier to fund multiple accounts that way